libs/vertexai/tests/integration_tests/test_vectorstores.py

```diff
@@ -41,6 +41,7 @@
     VectorSearchVectorStore,
     VectorSearchVectorStoreDatastore,
 )
+from tests.integration_tests.conftest import _DEFAULT_EMBEDDINGS_MODEL_NAME
```

Comment:
nits: that's seems to much. we have a fixture, hence there's no need to define a constant on top of that